automated install/basic-install.sh

```diff
@@ -1015,10 +1015,12 @@ valid_ip() {
     local ip=${1}
     local stat=1
 
-    # One IPv4 element is 8bit: 0 - 256
-    local ipv4elem="(25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9][0-9]?|0)";
-    # optional port number starting '#' with range of 1-65536
-    local portelem="(#([1-9]|[1-8][0-9]|9[0-9]|[1-8][0-9]{2}|9[0-8][0-9]|99[0-9]|[1-8][0-9]{3}|9[0-8][0-9]{2}|99[0-8][0-9]|999[0-9]|[1-5][0-9]{4}|6[0-4][0-9]{3}|65[0-4][0-9]{2}|655[0-2][0-9]|6553[0-6]))?"
+    # One IPv4 element is 8bit: 0 - 255
+    # See https://tools.ietf.org/html/rfc1340
+    local ipv4elem="(25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[0-9]{1,2})";
+    # optional port number starting ':' with range of 0-65535
+    # See https://tools.ietf.org/html/rfc6346
+    local portelem="(:(6553[0-5]|655[0-2][0-9]|65[0-4][0-9]{2}|6[0-4][0-9]{3}|[1-5][0-9]{4}|[0-9]{1,4}))?";
```

Comment:
I have about the same complaint as above. This would also allow `0053`

Reply:
Should be fixed, I changed `[0-9]{1,4}` to `[1-9][0-9]{0,3}|0`.